firebase_admin/db.py

```diff
@@ -101,6 +153,23 @@ def parent(self):
             return Reference(client=self._client, segments=self._segments[:-1])
         return None
 
+    def build_headers(self, token=None):
```

Comment:
See if we can remove this altogether (see my comment on _sseclient). If we must keep this, please rename to _build_headers() so it's treated as an internal method.

Reply:
This `build_headers()` is tested to be unnecessary just like your comment on `_sseclient`. So I will remove it from all subsequent code.